### PR TITLE
added a queue test case

### PR DESCRIPTION
**Pull request by DarrienKaelum**
Added a queue test case.
Added file changes from `kw/DiscordBotAndDBIntegration`

### CODE REVIEW

**Comment by jeffzhkw** [comment]
LGTM

File: commands/commands.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import path from 'node:path';
+
+//dynamic import of commands
+const all_commands = new Collection();
+const foldersPath = path.join(path.resolve(), 'commands/utils');
+const commandFolders = fs.readdirSync(foldersPath);
+
+console.log(`Loading commands from ${commandFolders}`);
+
+for (const commnadFilename of commandFolders) {
```
Comment (Thaileaf) [inline commands/commands.js:12]: commandFilename should be commandFilename

File: commands/commands.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const filePath = path.join(foldersPath, commnadFilename);
+
+  await import(filePath).then(command => {
+    //console.log(`Loaded command ${commnadFilename}`);
```
Comment (Thaileaf) [inline commands/commands.js:16]: Could potentially add these as separate debug fuctions, otherwise remove comments

File: commands/utils/queue.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+
+export default {
+  data: new SlashCommandBuilder()
+    .setName('queue')
+    .setDescription(
+      'For testing purposes, prints some user data. In the future will add a User to queue'
+    ),
+  async execute(interaction) {
+    console.log(interaction);
```
Comment (Thaileaf) [inline commands/utils/queue.js:10]: Functionality Looks good, potentially print more to discord to inform user about the queue

File: index.js [hunk 4]
```diff
@@ -1,33 +1,26 @@
...
+import all_commands from './commands/commands.js';
 import dbo from './src/server/db/conn.js';
 
+// using .env file to store sensitive information
 config();
-
 const TOKEN = process.env.DISCORD_TOKEN; // Discord Bot token obtained from Discord Developer Portal
 const GUILD_ID = process.env.GUILD_ID; // ID of the server where this bot will be used
 const CLIENT_ID = process.env.CLIENT_ID; // Client ID obtained from Discord Developer Portal
 const port = process.env.PORT || 5000;
```
Comment (Thaileaf) [inline index.js:11]: Some error handling code would be good here if it can't use port.

**Review by Thaileaf — COMMENTED** [review]
A couple of places we can do error handling and code clean up in the discord components

**Pull request merged**